### PR TITLE
Update the doc and code following PyCQA migration

### PR DESCRIPTION
**Pull request by Pierre-Sassoulas**
## Type of Changes

|     | Type                   |
| --- | ---------------------- |
| ✓   | :scroll: Docs          |

## Description

Refs https://github.com/PyCQA/meta/issues/54

### CODE REVIEW

**Comment by Pierre-Sassoulas** [comment]
Readthedoc and pre-commit should be fixed, I'm still working on re-syncing codecov (will finish tonight), but I don't understand the issue with python 3.11 tests not launching.

**Comment by Pierre-Sassoulas** [comment]
And ``codecov/codecov-action@*``

File: doc/faq.rst [hunk 1]
```diff
@@ -109,3 +109,18 @@ localized using the following rules:
...
+
+Why did pylint migrate from the PyCQA ?
+---------------------------------------
+
+One of our frequent contributor asked a question about ``pyproject.toml``
+support in flake8 and was banned without discussion. Discussions about it
+with the PyCQA admins revealed an irreconcilable approach to moderation.
+The two most active pylint contributors were demoted in the aftermath. As a
+result we were no longer able to administrate or moderate our own project
+effectively.
```
[the remark this answers is not in the document]
Comment (DanielNoord) [reply doc/faq.rst:121]: Agreed! I would even be okay with not mentioning the policies and just say: "Github Organisations share a lot of settings that are sometimes better set per project. By creating a new organisation we can control these better for the pylint associated projects".

In 2 years time nobody will probably care about the specifics anymore.

[the remark this answers is not in the document]
Comment (Pierre-Sassoulas) [reply doc/faq.rst:121]: The fact that me and PCManticore were both demoted  and that we could not administrate our own repo felt important, but we might also not add anything to the FAQ at all if we want to be super consensual. It's doubtless that it's interesting for a lot of persons anyway. Imo the important doc addition was #8329. I can remove the commit altogether

**Comment by Pierre-Sassoulas** [comment]
I'd like to reinstate readthedoc integration before merging, I'm looking into it.

**Comment by Pierre-Sassoulas** [comment]
Regarding right management, I realized that "member" in "pylint-dev" have no right (except seeing private repositories in the org, and we don't (won't?) have any). I'm wondering if we should:

- Give triage right everywhere to all member ?
- Add triager in each repositories ?

``pytest-dev`` is giving contributor right to everything on the first merged PR, which feels nice and makes me want to simplify thing and give triage right to member. Thought ?

**Comment by DanielNoord** [comment]
> I'd like to reinstate readthedoc integration before merging, I'm looking into it.

Should be fixed now!



> Regarding right management, I realized that "member" in "pylint-dev" have no right (except seeing private repositories in the org, and we don't (won't?) have any). I'm wondering if we should:
> 
> * Give triage right everywhere to all member ?

Let's not. Let's keep this a bit more fine-grained, or make the `member` more of a top-level thing with maintainers/triagers of individual projects not being `members` of the full organisation immediately.

> * Add triager in each repositories ?

Yes. If we start adding more plugins it will be good to have fine-grained access levels.


> * (Add member as a "honorary status" for 1 PR merged or 5 issues opened ?)

That limits us in what we can do with rights.

> `pytest-dev` is giving contributor right to everything on the first merged PR, which feels nice and makes me want to simplify thing and give triage right to member. Thought ?

This would be easier, but I can see how not everybody might want to add their project to this org if there are 20+ people with global triage rights.

**Comment by Pierre-Sassoulas** [comment]
All right, for the moment "member" is honorary (but all current members would be at least triager) and I'm going to add person one by one with actual rights in pylint/astroid. This feel like a whole discussion, let's not mix it with the current doc change..

**Comment by Pierre-Sassoulas** [comment]
Thank you for fixing readthedoc btw !

**Pull request merged**